.gitignore

```diff
@@ -27,3 +27,7 @@ docs/src
 _site
 .sass-cache
 .jekyll-metadata
+
+# Junk
+.DS_Store
+
```

Comment:
Someone is bound to come on the project and commit some .DS_Store files. I think it would just save headaches to add it here instead of having to discuss get someone to fix it later.